src/esp/bindings/PhysicsBindings.cpp

```diff
@@ -84,6 +84,42 @@ void initPhysicsBindings(py::module& m) {
           "motor_type", &JointMotorSettings::motorType,
           R"(The type of motor parameterized by these settings. Determines which parameters to use.)");
 
+  // ==== enum RigidConstraintType ====
+  py::enum_<RigidConstraintType>(m, "RigidConstraintType")
+      .value("PointToPoint", RigidConstraintType::PointToPoint)
+      .value("Fixed", RigidConstraintType::Fixed);
+
+  // ==== struct object RigidConstraintSettings ====
+  py::class_<RigidConstraintSettings, RigidConstraintSettings::ptr>(
+      m, "RigidConstraintSettings")
+      .def(py::init(&RigidConstraintSettings::create<>))
+      .def_readwrite("constraint_type",
+                     &RigidConstraintSettings::constraintType,
+                     R"(The type of constraint described by these settings.)")
+      .def_readwrite(
+          "max_impulse", &RigidConstraintSettings::maxImpulse,
+          R"(The maximum impulse applied by this constraint. Should be tuned relative to physics timestep.)")
+      .def_readwrite("object_id_a", &RigidConstraintSettings::objectIdA,
+                     R"(The id of the first object. Must be >=0.)")
```

Comment:
Perhaps mention here that any mixed-type constraints the AO be objectIdA, if this still holds.